### DISCUSSION TITLE
I can't see designer in Blend for VS...

### DISCUSSION THREAD

**Discussion by c00012** [AvaloniaUI/Avalonia#10134] [Q&A] [answered]
Hello,

I created an Avalonia with MVVM project to study Avalonia.
In VS 2022, I could see the designer in it like this:
![ice_screenshot_20230130-221623](https://user-images.githubusercontent.com/11556229/215498035-54c2503e-2b71-4aa0-9beb-f8d863b2b2f1.png)
but when I open the project in blend for VS, I couldn't see designer like this:
![ice_screenshot_20230130-221836](https://user-images.githubusercontent.com/11556229/215498260-69801eb2-2006-4064-9491-9fbbfc0257dd.png)

I reinstalled extensions and nuget packages for Avalonia but it didn't work.
if someone knows the solution for this, please let me know.

**Comment by maxkatz6** [comment, marked as answer]
Microsoft Blend isn't supported.

**Reply by c00012** [reply]
Really? then I can design Avalonia UI in VS only?

**Reply by maxkatz6** [reply]
You can't "design" it, you can preview the app.
Also, Rider supports previewer as well.

**Reply by c00012** [reply]
Ok, I got it. thanks for the answer.

**Comment by joer33304** [comment]
which is said. The ONLY thing missing for avalonia to be perfect is a visual designer
